app/assets/javascripts/application.js

```diff
@@ -14,3 +14,7 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+$(function() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/conta-srdr/forsythia/file/2137539563/issues/source?bid=3139784&fileBranchId=3148684#l18)